core/src/main/java/org/jenkins/ui/icon/IconSet.java

```diff
@@ -597,6 +597,7 @@ private static void initializeSVGs() {
         translations.put("icon-user", "symbol-people");
         translations.put("icon-undo", "symbol-undo");
         translations.put("icon-redo", "symbol-redo");
+        translations.put("icon-document-properties", "symbol-parameters");
```

Comment:
Unsure about this one. There seems to be quite the semantic difference between the two.

Reply:
This is no original tango svg, but a remixed one based on tango components. 

A more matching ionicon is the plain wrench, going off the tango base:
![Screenshot 2022-07-16 at 17 38 05](https://user-images.githubusercontent.com/13383509/179361682-5ea55bb4-076d-414f-a0a3-6ec69ee8d799.png)
But the proposed one looks fine for build parameters to me.

Reply:
>  But the proposed one looks fine for build parameters to me.

Agree per https://github.com/jenkinsci/jenkins/pull/6862#discussion_r922680390, just this replacement doesn't look that useful.